completions/_vagrant

```diff
@@ -14,6 +14,13 @@ _vagrant () {
 		fi
 		_describe -t commands "vagrant $1" vms -V1
 	}
+	getboxes() {
+        	local -a boxes
+        	boxes=(
+			$(print -l $(basename ~/.vagrant.d/boxes/*(/)))
```

Comment:
`print -l $(basename ~/.vagrant.d/boxes/*(/))` Should only be looking at directories in ~/vagrant.d/boxes/